msteams-platform/concepts/capabilities-overview.md

```diff
@@ -21,12 +26,22 @@ Your Teams app have one or all of the following core capabilities:
 
 Your app can also take advantage of advanced capabilities, such as the [Microsoft Graph API for Teams](https://docs.microsoft.com/graph/teams-concept-overview).
 
-See the following illustration to get an idea which capabilities would provide the features you want in your app.
+The following illustration gives you an idea of which capabilities will provide the features you want in your app.
 
 :::image type="content" source="../assets/images/capabilities-overview.png" alt-text="Mind map illustrating what Teams app capabilities are.":::
 
-## Doing what's best for your users
+The core capabilities together with the entry points enable you to create a collaboration hub that can provide a native experience to your users. 
```

Comment:
@KirtiPereira Suggest removing this sentence or revising since Teams itself is a collaboration hub.